councilmatic_core/management/commands/refresh_pic.py

```diff
@@ -56,24 +56,21 @@ def _get_urls(self):
         with connection.cursor() as cursor:
 
             query = '''
-                SELECT DISTINCT url from 
-                councilmatic_core_billdocument as b_doc
-                JOIN change_bill 
-                ON change_bill.ocd_id=b_doc.bill_id
-                WHERE document_type='V'
```

Comment:
note that ocd does not include a good way to store the A/V values indicating whether a document is a version. we use this to filter this query and one in convert_attachment_text. i'm not sure what the consequences of omitting them will be.

Reply:
actually nvm